### PR TITLE
Exclude hardware wallets from TTS tracking

### PR DESCRIPTION
**Pull request by walmat**
Fixes APP-1970

## What changed (plus any additional context for devs)
Disables time tracking functionality if we're using a hardware wallet. This prevents skewed data where time to sign could take significantly longer for a hardware wallet.

## Screen recordings / screenshots
n/a

## What to test
- does it disable for hardware wallets?
- is it enabled for non-hardware wallets?

### CODE REVIEW

File: src/state/performance/performance.ts [hunk 1]
```diff
@@ -23,6 +24,12 @@ interface PerformanceTrackingState {
   };
 }
 
+export function isEnabled() {
+  const isHardwareWallet = store.getState().wallets.selected?.deviceId;
```
Comment (brunobar79) [inline src/state/performance/performance.ts:28]: Just checking you're 100% sure that store.getState().wallets is never null or undefined

**Review by brunobar79 — APPROVED** [review]

File: src/state/performance/performance.ts (hunk 1 above)
Comment (walmat) [reply src/state/performance/performance.ts:28]: good point I'll null guard it

Comment (walmat) [reply src/state/performance/performance.ts:28]: Actually looking at the type definition, it should be always defined. `wallets.wallets` could be `null`

**Comment by brunobar79** [comment]
Launch in [simulator](http://localhost:29070/install/ios?virtual=https://app-team.p.rainbow.me/@matthew/APP-1970/704d7d4600a7251097dd6b247b35afab33d70951.app.zip) or [device](http://localhost:29070/install/ios?physical=https://app-team.p.rainbow.me/@matthew/APP-1970/704d7d4600a7251097dd6b247b35afab33d70951.ipa) for 704d7d4600a7251097dd6b247b35afab33d70951

**Review by greg-schrammel — APPROVED** [review]

**Pull request merged**